framework/interface/api_handlers.py

```diff
@@ -63,18 +77,32 @@ def get(self, target_id=None, plugin_group=None, plugin_type=None, plugin_code=N
             ## Get test groups as well, for names and info links
             test_groups = {}
             for test_group in self.get_component("db_plugin").GetAllTestGroups():
-                if test_group['code'] in results:
-                    test_group["mapped_code"] = test_group["code"]
-                    test_group["mapped_descrip"] = test_group["descrip"]
-                    if mappings:
-                        try:
-                            test_group["mapped_code"] = mappings[test_group['code']][0]
-                            test_group["mapped_descrip"] = mappings[test_group['code']][1]
-                        except KeyError:
-                            pass
-                    test_groups[test_group['code']] = test_group
-            test_groups = collections.OrderedDict(sorted(test_groups.items()))
-            self.write(test_groups)
+                test_group["mapped_code"] = test_group["code"]
+                test_group["mapped_descrip"] = test_group["descrip"]
+                if mappings:
+                    try:
+                        test_group["mapped_code"] = mappings[test_group['code']][0]
+                        test_group["mapped_descrip"] = mappings[test_group['code']][1]
+                    except KeyError:
+                        pass
+                test_groups[test_group['code']] = test_group
+
+            dictToReturn = {}
```

Comment:
Avoid using CamelCase - it is not pythonic. Use _ and lowercase characters in case of variable names and functions. (See snakecase for reference)